master/getting-started/kubernetes/tutorials/app-layer-policy/index.md

```diff
@@ -0,0 +1,298 @@
+and occasionally attackers can find ways to trick applications into giving up secrets they should
+not.  For example, the [Heartbleed] vulnerability in OpenSSL allowed attackers to trick an affected
+application into reading out portions of its memory, compromising private keys (among other
+confidential information).
+
+Let's simulate an attacker who has stolen the private keys of another pod.  Since the keys are
+stored as Kubernetes secrets, we won't exploit a vulnerability in a service, but instead just mount
+the secret in a pod that will simulate an attacker.
+
+If you are still `exec`'d into the customer pod, exit out or open a new terminal tab (we will return
+the to the customer pod later). 
+
+```bash
+kubectl apply -f \
+{{site.url}}/{{page.version}}/getting-started/kubernetes/tutorials/app-layer-policy/manifests/20-attack-pod.yaml
+```
+
+Take a look at the [`20-attack-pod.yaml` manifest in your browser](manifests/20-attack-pod.yaml).
+It creates an `ubuntu` pod and mounts `istio.summary` secret.  This will allow us to masquerade as
+if we were the `summary` service, even though this pod is not run as that service account.  Let's
+try this out.  First, `exec` into the pod.
+
+    kubectl exec -ti attack-<fill in pod ID> bash
+
+Next, install the `curl` utility to initiate HTTP connections from the command line.
+
+    apt update && apt install -y curl
+    
+Now, we will attack the database.  Instead of listing the contents like we did before, let's try 
+something more malicious, like changing the account balance with a `PUT` command.
+
+    curl -k https://database:2379/v2/keys/accounts/519940/balance -d value="10000.00" -XPUT --key /etc/certs/key.pem --cert /etc/certs/cert-chain.pem
+
+Unlike when we did this with the customer web pod, we do not have the Envoy to handle encryption, so
+we have to pass an `https` URL, the `--key` and `--cert` parameters to `curl` to do the
+cryptography.
+
+Return to your web browser and refresh to confirm the new balance.
+
+#### Policy
+
+We can mitigate both of the above deficiencies with a Calico policy.
+
+    wget {{site.url}}/{{page.version}}/getting-started/kubernetes/tutorials/app-layer-policy/manifests/30-policy.yaml
+    calicoctl create -f 30-policy.yaml
+
+> **Note**: You can also
+> [view the manifest in your browser](manifests/30-policy.yaml){:target="_blank"}.
+{: .alert .alert-info}
+
+Let's examine this policy piece by piece.  It consists of 3 policy objects, one for each microservice.
+
+    apiVersion: projectcalico.org/v3
+    kind: GlobalNetworkPolicy
+    metadata:
+      name: customer
+    spec:
+      selector: app == 'customer'
+      ingress:
+        - action: Allow
+          http:
+            methods: ["GET"]
+      egress:
+        - action: Allow
+
+This policy protects the customer web app.  Since this application is customer facing, we do not
+restrict what can communicate with it.  We do, however, restrict that only HTTP `GET` requests are
```

Comment:
The construction of the last sentence is a little problematic. I would suggest perhaps something more like "We do, however, restrict its communications to HTTP `GET` requests."